app/src/main/kotlin/com/santaclose/app/restaurant/resolver/dto/RestaurantAppQueryResolver.kt

```diff
@@ -0,0 +1,31 @@
+package com.santaclose.app.restaurant.resolver.dto
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.server.operations.Query
+import com.santaclose.app.auth.directive.Auth
+import com.santaclose.app.restaurant.service.RestaurantAppQueryService
+import com.santaclose.lib.entity.appUser.type.AppUserRole
+import com.santaclose.lib.logger.logger
+import com.santaclose.lib.web.error.toGraphQLException
+import com.santaclose.lib.web.toLong
+import javax.validation.Valid
+import org.springframework.stereotype.Component
+
+@Component
+class RestaurantAppQueryResolver(private val restaurantAppQueryService: RestaurantAppQueryService) :
+  Query {
+  val logger = logger()
+
+  @Auth(AppUserRole.USER)
+  @GraphQLDescription("맛집 상세 조회")
+  fun restaurantDetail(
+    @Valid input: RestaurantDetailAppInput,
```

Comment:
여기는 input 대신 ID 하나만 받게하면 어떨까요?
input 객체안에 딱 하나의 필드만 있는 상황이라 객체로 감쌀 필요가 없는거 같아요.
만약 바꾼다면 클라이언트에 요청 시 아래처럼 되서 더 간결할거 같습니다.

```graphql
query {
  restaurantDetail(id: "123") {
    ... 필드
  }
}
```

```suggestion
    id: ID,
```